File: frontend/src/app/models/NewSubject.ts
export class NewSubject {
    idnewsubject: number;
    name: string;
    idprofessor: number;
  
    constructor(idnewsubject: number, name: string, idprofessor: number) {
      this.idnewsubject = idnewsubject;
      this.name = name;
      this.idprofessor = idprofessor;
    }
  
    getIdnewsubject(): number {
      return this.idnewsubject;
    }
  
    setIdnewsubject(idnewsubject: number): void {
      this.idnewsubject = idnewsubject;
    }
  
    getName(): string {
      return this.name;
    }
  
    setName(name: string): void {
      this.name = name;
    }
  
    getIdprofessor(): number {
      return this.idprofessor;
    }
  
    setIdprofessor(idprofessor: number): void {
      this.idprofessor = idprofessor;
    }
  }

File: backend/src/main/java/com/example/backend/models/NewSubject.java
package com.example.backend.models;

public class NewSubject {
    private int idnewsubject;
    private String name;
    private int idprofessor;
    public NewSubject(int idnewsubject, String name, int idprofessor) {
        this.idnewsubject = idnewsubject;
        this.name = name;
        this.idprofessor = idprofessor;
    }
    public int getIdnewsubject() {
        return idnewsubject;
    }
    public void setIdnewsubject(int idnewsubject) {
        this.idnewsubject = idnewsubject;
    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public int getIdprofessor() {
        return idprofessor;
    }
    public void setIdprofessor(int idprofessor) {
        this.idprofessor = idprofessor;
    }
    
}
